src/sonic-yang-mgmt/yang-models/sonic-port.yang

```diff
@@ -0,0 +1,90 @@
+module sonic-port{
+
+	namespace "http://github.com/Azure/sonic-port";
+	prefix port;
+
+	import ietf-yang-types {
+		prefix yang;
+	}
+
+	import ietf-inet-types {
+		prefix inet;
+	}
+
+	import sonic-head {
+		prefix head;
+		revision-date 2019-07-01;
+	}
+
+	organization "Linkedin Corporation";
+
+	contact "lnos_coders@linkedin.com";
+
+	description "PORT yang Module for SONiC OS";
+
+	revision 2019-07-01 {
+		description "First Revision";
+	}
+
+	container sonic-port{
+		container PORT {
+
+			description "PORT part of config_db.json";
+
+			list PORT_LIST {
+
```

Comment:
Missing index & pfc_asym

Reply:
Added index. pfe_asym is not in BCRM models too as of now, I think feature owner can add this later.